flaskr/gs_mock.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+from flask import request, jsonify
+import numpy as np
+from pydantic import BaseModel, Field
+from typing import List, Optional
+
+
+class MockRequest(BaseModel):
+    start: datetime
+    end: datetime
+    delta_minutes: int
+    seed: Optional[int] = Field(default=None)
```

Comment:
Could you please add short descriptions of what delta and seed mean in this context?